app/models/template_kind.rb

```diff
@@ -24,6 +24,21 @@ def self.default_template_labels
     }
   end
 
+  def self.template_description
```

Comment:
Nitpick: To reflect the `default_template_labels` method name above this could be `default_template_descriptions`.